src/Umbraco.Web/PropertyEditors/MediaPickerPropertyEditor.cs

```diff
@@ -8,17 +8,17 @@ namespace Umbraco.Web.PropertyEditors
 {
     /// <summary>
     /// Represents a media picker property editor.
-    /// Marked as Deprecated as best to use the NEW Media Picker aka MediaPicker3
+    /// Nameed "(legacy)" as it's best to use the NEW Media Picker aka MediaPicker3
```

Comment:
@nul800sebastiaan a minor typo in "Named" 😁

Reply:
```suggestion
    /// Named "(legacy)" as it's best to use the NEW Media Picker aka MediaPicker3
```